src/templates/CustomTypeTemplate.java

```diff
@@ -0,0 +1,131 @@
+{%- let package_name = config.package_name() %}
+{%- match config.custom_types.get(name.as_str())  %}
+{%- when None %}
+{#- Define the type using typealiases to the builtin #}
+{%- let ffi_type_name=builtin|ffi_type|ffi_type_name_by_value %}
+
+ package {{ package_name }};
+
+// Newtype
```

Comment:
This comment is going to go into every generated instance and isn't doing much. Actually a bit surprised there wasn't any `docstring` related generation for the types in the Kotlin version.